torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -1022,7 +1030,44 @@ def render(  # type: ignore[override]
                 op = self.swap_XW(op)
                 should_swap_xw = True
 
-        instance_definition, instance_type = self._define_gemm_instance(op)
+        if epilogue_nodes:
+            evt_read_names, evt_write_names, buffer_renames, evt_py_code = (
+                CutlassEVTCodegen.ir_to_evt_python_code(Y.get_name(), epilogue_nodes)
+            )
+            read_names = OrderedSet(evt_read_names) - OrderedSet(evt_write_names)
+            write_names = OrderedSet(evt_write_names)
+
+            name_to_buffer = V.graph.name_to_buffer | V.graph.graph_inputs
+            epilogue_inputs = [name_to_buffer[name] for name in read_names]
+            epilogue_outputs = [name_to_buffer[name] for name in write_names]
+
+            evt_name, evt_args, evt_code = self._render_evt(
+                op,
+                evt_py_code,
+                evt_read_names,
+                evt_write_names,
+                buffer_renames,
+                Y.get_layout().dtype,
+                W.get_layout().dtype,
+            )
+        else:
+            epilogue_inputs = []
+            epilogue_outputs = []
+            evt_name = None
+            evt_args = f"{{ElementComputeEpilogue({self.alpha}), ElementComputeEpilogue({self.beta})}}"
+            evt_code = ""
+
+        kernel_call_signature = kernel.def_kernel(
+            inputs=inputs,  # type: ignore[arg-type]
+            outputs=[Y],
+            epilogue_inputs=[],
+            epilogue_outputs=epilogue_inputs + epilogue_outputs,
```

Comment:
Fix this, decide what we want to do here, the issue is we need to put const dtype *  when generating the arguments, perhaps just pass this information to _render_evt and trace?